README.md

```diff
@@ -5,53 +5,71 @@
 
 This workspace contains the Geo Engine crates.
 
-# Development
+## Development
```

Comment:
Dependencies/lints etc. should be under "Development". So please indent them as well

Reply:
d52a58c